Cowles.js

```diff
@@ -0,0 +1,316 @@
+{
+	"translatorID": "951c027d-74ac-47d4-a107-9c3069ab7b48",
+	"label": "Cowles",
```

Comment:
"Cowles Foundation" seems like the more appropriate label & filename

Reply:
I changed the filename, can you confirm that I can delete the old file? I guess it will delete all comments but I did not know how else to do it